spec/post_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
Please add a test for `#recent_five_comments` on the Post model as it's a requirement to test custom methods in the model.

Reply:
Dear @SajjadAhmad14 
Thank you for the feedback. I have realized the requirements and i am doing my best to implement them.